### ISSUE TITLE
flame1d production driver fails to run on Lassen

### ISSUE THREAD

**Issue by MTCam** [illinois-ceesd/mirgecom#651]
This error occurs for current mirgecom@production, presumably due to changes in upstream packages.  To reproduce:

1. install mirgecom@production 
2. clone github.com:/illinois-ceesd/drivers_flame1d
3. run flame1d/timing_run test: python -m mpi4py ./flame1d.py -i run_params.yaml --lazy

After initialization and after a little bit of compiling:
```
CUDA_ERROR_INVALID_PTX: a PTX JIT compilation failed
ERROR:  One or more process (first noticed rank 0) terminated with signal 6
```

**Comment by matthiasdiener** [comment]
Which actx is that?

**Comment by MTCam** [comment]
We're back baby!  Nice improvement with fusion.  Fixed by making #621 a `production-feeder`.

<img width="998" alt="Screen Shot 2022-04-29 at 2 38 02 PM" src="https://user-images.githubusercontent.com/8116930/166058419-fcaa2e68-005f-4533-9ab4-1e1aab78d437.png">
